Added instructions on how to clear voice commands (#442)
smartdevicelink/sdl_app_library_guides: docs/Speech and Audio/Setting Up Voice Commands/index.md

```
!@

## Deleting Voice Commands
To delete previously set voice commands, you just have to set an empty array to the `voiceCommands` array on the screen manager.

@![iOS]
##### Objective-C
```objc

> sdlManager.screenManager.voiceCommands = [];

Reviewer: ```suggestion
self.sdlManager.screenManager.voiceCommands = [];
```

```
!@

## Deleting Voice Commands

> To delete previously set voice commands, you just have to set an empty array or `nil` to the `voiceCommands` array on the screen manager.

Reviewer: ```suggestion
To delete previously set voice commands, you just have to set an empty array to the `voiceCommands` array on the screen manager.
```

@![iOS]
##### Objective-C
```objc
sdlManager.screenManager.voiceCommands = [];

> sdlManager.screenManager.voiceCommands = nil;

Reviewer: ```suggestion
```

Reviewer: I was wrong. You can't set `nil` for `voiceCommands` and an error will be thrown at compile time

```

##### Swift
```swift
sdlManager.screenManager.voiceCommands = []

> sdlManager.screenManager.voiceCommands = nil

Reviewer: ```suggestion
```

```
!@

@![android, javaSE, javaEE]
```java
// TODO: java code snippets
```
!@

@![javascript]
```js
// TODO: javascript code snippets
```
!@

!!! NOTE

> Setting voice command strings consisted of whitespace characters only will be considered invalid(e.g.  " ") and your request will be aborted.

Reviewer: ```suggestion
Setting voice command strings composed only of whitespace characters will be considered invalid (e.g.  `" "`) and your request will be aborted by the module.
```
